src/NzbDrone.Core/Datastore/Migration/227_update_restrictions_to_release_profiles.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Dapper;
+using FluentMigrator;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(227)]
+    public class update_restrictions_to_release_profiles : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Rename.Table("Restrictions").To("ReleaseProfiles");
+
+            Alter.Table("ReleaseProfiles").AddColumn("Name").AsString().Nullable().WithDefaultValue(null);
+            Alter.Table("ReleaseProfiles").AddColumn("Enabled").AsBoolean().WithDefaultValue(true);
+            Alter.Table("ReleaseProfiles").AddColumn("IndexerId").AsInt32().WithDefaultValue(0);
+            Delete.Column("Preferred").FromTable("ReleaseProfiles");
+
+            Execute.WithConnection(ChangeRequiredIgnoredTypes);
+
+            Execute.Sql("DELETE FROM \"ReleaseProfiles\" WHERE \"Required\" == '[]' AND \"Ignored\" == '[]'");
```

Comment:
This would be a more native to FluentMigrator way to achieve this 

```suggestion
            Delete.FromTable("ReleaseProfiles").Row(new { Required = "[]", Ignored = "[]" });
```

Reply:
I guess this also works fine for postgres dbs? I'm not too familiar with FluentMigrator, but if it is, I'll just commit your suggestion.

Reply:
I agree with that actually Bogdan!

Reply:
The fixture is probably a good call here, @ricci2511 feel free to take a stab at that (I'm surprised there wasn't one on Sonarr given the issues they had with this migration). If you want me to take a stab I may have some time today.

Reply:
I'll give it a try tomorrow, if anything i'll hop on the discord

Reply:
Sorry for the delay. I just pushed a fixture for the migration. I specifically focused on the `required` and `ignored` columns, so let me know if I should add any other fixtures.